src/config/app.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Support\Facades\Facade;
+use Illuminate\Support\ServiceProvider;
+
+$providers = ServiceProvider::defaultProviders()->merge([
+    ExampleApp\Providers\AppServiceProvider::class,
+    ExampleApp\Providers\EventServiceProvider::class,
+    ExampleApp\Providers\RouteServiceProvider::class,
```

Comment:
Are you sure this would work? With no `\` at the beginning of classes? Anyway, I would import them.

Reply:
imported